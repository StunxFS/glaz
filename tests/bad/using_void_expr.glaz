fn x() {}

fn main() {
  _ = x();
}
