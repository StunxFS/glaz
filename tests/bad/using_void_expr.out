tests/bad/using_void_expr.glaz:4:8: error: invalid use of void expression
     | 
   4 |   _ = x();
     |       ^^^
glaz error: could not compile `using_void_expr`, aborting due to previous error.
